Files/Views/Pages/PropertiesSecurity.xaml

```diff
@@ -95,15 +95,25 @@
                     Style="{StaticResource PropertyName}"
                     Text="Group or user names" />
 
-                <CommandBar
+                <StackPanel
```

Comment:
@yaichenbaum note that there is another command bar you may wish to change

Reply:
@gave92 Taken care of, thanks!